client/latis.py

```diff
@@ -4,6 +4,20 @@
 import urllib.parse
 
 
+def data(baseUrl, latis3, dataset, returnType,
```

Comment:
I'm still waffling on a single function with lots of args vs specific functions like `downloadCsv` or `readNumpy`. I suppose we could have the former with the latter as shortcuts.

Reply:
Did we ever reach a conclusion on new naming conventions? Currently sticking with using arguments to determine Numpy/Pandas or file format.